power-supply/power_supply.cpp

```diff
@@ -676,14 +660,15 @@ void PowerSupply::updateInventory()
         {
         }
 
+#ifdef UCD90160_DEVICE_ACCESS
```

Comment:
UCD90160_DEVICE_ACCESS is not used to check UCD or mihawk. You need to create another macro.

Reply:
@alvintpwang I will fix it.

Reply:
You're trying to skip these on mihawk?

Reply:
@spinler 
Because mihawk psu does not support ccin (input-history), skip "ccin" part is required for mihawk.
I will fix this part.